apps/frontend/src/app/not-found.tsx

```diff
@@ -1,34 +1,42 @@
-import Link from "next/link"
-
 import { UabcHeaderText } from "@/components/Composite/UabcHeaderText"
 import { UabcLogoNotFound } from "@/components/Composite/UabcLogoNotFound"
-import { Button } from "@/components/Generic/ui/button"
+import { Button } from "@repo/ui/components/Button"
+import { Heading } from "@repo/ui/components/Heading"
+import { Center, Container, Text, VStack } from "@yamada-ui/react"
+import Link from "next/link"
 
 export const metadata = {
   title: "Not Found - UABC Booking Portal",
 }
 
 const NotFoundPage = () => {
   return (
-    <div className="flex h-dvh w-dvw flex-col items-center justify-evenly overflow-hidden bg-background">
+    <Container centerContent height={"100dvh"} justifyContent={"space-evenly"}>
```

Comment:
Can we use `justifyContent="center"` here?
```suggestion
    <Container centerContent h="100dvh" justifyContent="center">
```